docs/env_vars.md

```diff
@@ -39,6 +39,11 @@ wlroots reads these environment variables
 
 * *WLR_X11_OUTPUTS*: when using the X11 backend specifies the number of outputs
 
+## gles2 renderer
+
+* *WLR_RENDERER_ALLOW_SOFTWARE*: allows the gles2 renderer to use software
```

Comment:
Note to self: this is a GLES2-specific variable, but it doesn't contain "GLES2" in the name. I'd like to implement a similar check in other non-GLES2 renderers such as Vulkan.